### DISCUSSION TITLE
Top navbar tabs AND expandable sidebar nav sections?

### DISCUSSION THREAD

**Discussion by rmscode** [squidfunk/mkdocs-material#6455] [Q&A] [answered]
I was going nuts earlier trying to figure out how to make my sidebar navigation sections expandable. I had seen it before, so I commented out all my `navigation` settings and there it was . . . the sidebar nav with expandable/collapsible sections. After enabling `navigation.tabs` again, it was gone. 

Is there a way to have _both_?

I would like this:
![1](https://github.com/squidfunk/mkdocs-material/assets/26286185/67e3f22f-0cc2-4dcd-b594-732242f02ed9)

AND this:
![2](https://github.com/squidfunk/mkdocs-material/assets/26286185/7c1446a3-8a41-4c94-b949-f8cb15393032)

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello,
once you enable `navigation.tabs` level 1 navigation sections become tabs on the website. Then all other navigation sections are rendered on the sidebar. That's the purpose of tabs, to change the context of the displayed content. Therefore, if you enable tabs and Hardware is it's own section there isn't much sense in adding other sections like Networking / Software to the sidebar visible on Hardware.

As far as I know there is no config option to achieve what you ask for.
Only thing you could do, would be to use theme overrides and override the `nav.html` or `nav-item.html` partial.
https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme

**Comment by alexvoss** [comment]
I am also not aware of a way to achieve what you are asking for, other than customizing the partials. The point of `navigation.tabs`, as Kamil said, is to hoist the top-level navigation to the tabs and leave a simpler navigation of each individual section on the left-hand side. I am not entirely sure why you would replicate the top level. 

The [FastAPI site](https://fastapi.tiangolo.com/tutorial/) is a good example how how it is meant to work. Before you consider changing partials, ask yourself if the approach you choose will really make sense in the long-run.

**Comment by rmscode** [comment]
@kamilkrzyskow & @alexjlockwood Thank you both for your replies. What you've said makes sense. I'm realizing I misunderstood how `navigation.tabs`/mkdocs worked when generating the navigation. I thought those sections were linked to the headers (#) of a document, but that's what the TOC is for. What I think I need to do is structure my directories and `mkdocs.yml` in a way that gives me what I'm looking for.

**Reply by kamilkrzyskow** [reply]
Perhaps you're looking for https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-integration
